frontend/src/api/api.tsx

```diff
@@ -4,8 +4,9 @@ import axios from 'axios';
 import { RequestError } from '../types/RequestError';
 import { TutorialOffer } from '../types/Tutorial';
 import { User } from '../types/User';
+import jsonFile from './url.json'
 
-const backendUrl = 'http://localhost:8080';
+const backendUrl = jsonFile.api_url;
```

Comment:
Warum kein `.env` Dateien nutzen?
Best practice ist soweit ich weiß das Vorgehen das zB bei [create-react-app](https://create-react-app.dev/docs/adding-custom-environment-variables/) beschrieben wird

Reply:
Siehe pr message

Reply:
Siehe [7f8b05a](https://github.com/FinnHuelsbusch/tutorensystem/pull/25/commits/7f8b05a1f6cc0bce2c1da4b52bd82228edcd71af)

Reply:
das .env file muss ins .gitignore

Reply:
@Luc500 Falls es funktioniert sag Bescheid

Reply:
Funktioniert
Na wunderbar, dann machen wir es so
Die Bennenungskonvention hatte ich nicht entdeckt

Reply:
@Luc500 👍 füge in das Readme vom Frontend Ordner und/oder in Discord noch eine Nachricht dazu dass jeder ein `.env`mit bestimmter Struktur braucht.
Auch den Hinweis, dass man jedes mal `npm start` neu ausführen muss bei Änderung der env Datei.

Reply:
Oder ein `.env.demo` das nur die Struktur enthält aber keine Werte und das mitgepusht wird